src/layouts/index.tsx

```diff
@@ -31,8 +31,8 @@ const Layout: React.FC<{ children: ReactNode }> = ({ children }) => {
         <EditorSideBar />
         {children}
         <TopBar />
-        <Footer />
       </Box>
+      {/* <Footer /> */}
```

Comment:
If this footer is not used anymore and no UI need to use it, can simply delete it with using and import.